src/views/RequiredSignoffs/utils/updateRequiredSignoffs.js

```diff
@@ -27,104 +26,93 @@ export default async params => {
     role => !currentRoles.includes(role.name)
   );
   let useScheduledChange = !isNewSignoff;
-  const [error] = await tryCatch(
-    Promise.all(
-      [].concat(
-        roles.map(async role => {
-          const extraData = role.sc
-            ? { sc_data_version: role.sc.data_version }
-            : {};
-          let skip = false;
 
-          originalRoles.forEach(value => {
-            const newSignoffsRequired = role.sc
-              ? role.sc.signoffs_required
-              : role.signoffs_required;
-            const originalSignoffsRequired = value.sc
-              ? value.sc.signoffs_required
-              : value.signoffs_required;
+  await Promise.all(
```

Comment:
Nit: Now that we're not checking for the error in this function, this function doesn't need to have the `async` signature nor `await`. `await` can be replaced with `return`.